packages/overlays/src/GlobalOverlayController.js

```diff
@@ -57,6 +57,14 @@ export class GlobalOverlayController extends BaseOverlayController {
     }
   }
 
+  _connectContent() {
```

Comment:
this should probably be called when a contentNode is passed to a new controller instance, because it means 'css' should be show hide mode, and if that's true, the content should be connected and hidden. Right now I'm doing it in the account select